config/leviton/vrpa1.xml

```diff
@@ -0,0 +1,41 @@
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/001D:0334:1805</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/leviton/dz15s.png</MetaDataItem> <!-- This item looks similar to the DZ15s-->
```

Comment:
Please add a copy - If someone changes dz15s then this device would be affected.